canvas_admin_tools/urls.py

```diff
@@ -4,10 +4,12 @@
 
 
 urlpatterns = [
+    url(r'^django-rq/', include('django_rq.urls')),
```

Comment:
Will the `staff_member_required` check in django-rq's views adequately protect it (e.g. from public attempts to delete jobs) while allowing administrators to access it? (I don't think I've seen one of our apps with a default Django /admin dashboard included, not sure how our users respond to `is_staff`.)